tests/dummy/app/routes/friend.js

```diff
@@ -0,0 +1,8 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+
+  model(params) {
+    return this.store.find('friend', params.friend_id);
+  }
+});
```

Comment:
i think this file is unnecessary, no? isn't it the default behavior for that route